src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/JndiAccessPolicyDAO.scala

```diff
@@ -170,6 +157,17 @@ class JndiAccessPolicyDAO(protected val directoryConfig: DirectoryConfig)(implic
     newPolicy
   }
 
+  override def listAccessPolicies(resourceTypeName: ResourceTypeName, userId: WorkbenchUserId): Future[Set[ResourceIdAndPolicyName]] = withContext { ctx =>
+    val attributes = ctx.getAttributes(userDn(userId), Array(Attr.memberOf))
+    val groupDns = getAttributes[String](attributes, Attr.memberOf).getOrElse(Set.empty).toSet
+
+    val policyDnPattern = s"(?i)${Attr.policy}=([^,]+),${Attr.resourceId}=([^,]+),${Attr.resourceType}=${resourceTypeName.value},$resourcesOu".r
```

Comment:
this bit was discussed a good amount during the verbal review- were you still going to explore an alternative?